packages/generator/src/sdk-metadata/code-samples.ts

```diff
@@ -29,3 +30,47 @@ export function genericCodeSample(): InstructionWithTextAndHeader {
     header: usageHeaderText
   };
 }
+
+export function importsCodeSample(
+  actionFunctionImport: VdmFunctionImport | VdmActionImport,
+  packageName: string
+): InstructionWithText {
+  return {
+    text: 'To consume the service via the pregenerated typed client library run the code snippet below. For more details about OData client libraries chose "OData Consumption Manual" from the "Helpful Links" menu.',
+    instructions: codeBlock`
+import { ${actionFunctionImport.name}${
+      actionFunctionImport.parametersTypeName
+        ? `, ${actionFunctionImport.parametersTypeName}`
+        : ''
+    } } from '${packageName}';
+
+${getParameterCodeSample(actionFunctionImport)}
+const resultPromise = ${actionFunctionImport.name}(${
+      actionFunctionImport.parametersTypeName ? 'parameter' : ''
+    }).execute({ destinationName:'myDestinationName' });
+`
+  };
+}
+
+function getParameterCodeSample(functionImport: VdmFunctionImport): string {
+  if (functionImport.parameters) {
+    return `const parameter: ${
+      functionImport.parametersTypeName
+    } = ${getActionFunctionParams(functionImport.parameters)};
+    `;
+  }
+  return '';
+}
+
+function getActionFunctionParams(parameters: VdmParameter[]): string {
```

Comment:
For this method a dedicated unit test would make sense to see the cases: below 2 above 2 parameters and that the string is correct.